tests/unit_tests/dashboards/commands/importers/v1/utils_test.py

```diff
@@ -49,8 +49,10 @@ def test_update_id_refs_immune_missing(  # pylint: disable=invalid-name
                 "101": {"filter_name": {"immune": [102, 103]}},
                 "104": {"filter_name": {"immune": [102, 103]}},
             },
+            "native_filter_configuration": [
+                {"scope": {"excluded": ["uuid1", "uuid2"]}}
```

Comment:
These should be IDs, not UUIDs.

Also, can we have a new test to check the logic, instead of piggybacking on `test_update_id_refs_immune_missing`?

Reply:
rookie mistake this is on import i will update accordingly 😅